### PR TITLE
Update seeders and add batch file to reindex

### PR DESCRIPTION
**Pull request by FireMartZ**

### CODE REVIEW

File: win/first-run.bat [hunk 1]
```diff
@@ -10,13 +10,23 @@ IF NOT EXIST %AppData%\Hush\hush.conf (
     echo addnode=stilgar.leto.net
     echo addnode=zdash.suprnova.cc
```
Comment (joshgentry) [inline win/first-run.bat:11]: This diff LGTM but we might want to remove this `supernova.cc` default node, as they're the ones hosting the conflicting seed?

File: win/reindex.bat [hunk 2]
```diff
@@ -0,0 +1 @@
+hushd.exe -reindex
```
Comment (joshgentry) [inline win/reindex.bat:1]: It's sad that we need this :(

File: win/first-run.bat (hunk 1 above)
Comment (FireMartZ) [reply win/first-run.bat:11]: I don't mind removing it.  AFAIK there is no hard evidence about suprnova being the root cause.

File: win/reindex.bat (hunk 2 above)
Comment (FireMartZ) [reply win/reindex.bat:1]: Until we find what cause all those people having to reindex, this little shortcut will save a lot of time explaining how to reindex on Discord.

Comment (leto) [reply win/reindex.bat:1]: :+1:

**Review by leto — APPROVED** [review]
Thanks, this looks good :+1:

**Pull request merged**